libbeat/keystore/keystore.go

```diff
@@ -73,6 +74,11 @@ type ListingKeystore interface {
 	List() ([]string, error)
 }
 
+// Provider for keystore
+type KeystoreProvider interface {
```

Comment:
nit; stuttering. The package is already named `keystore`, which makes the full symbol name read `keystore.KeystoreProvider`. Let's name the interface `keystore.Provider`.

Reply:
👍